### PR TITLE
Fix quoting

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

File: src/uri.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 const ALPHA = 'a-zA-Z';
 const DIGIT = '0-9';
-const GEN_DELIMS = "\\:\\/\\?\\#\\[\\]\\@"; // eslint-disable-line quotes
-const SUB_DELIMS = "\\!\\$\\&\\'\\(\\)\\*\\+\\,\\;\\="; // eslint-disable-line quotes
+const GEN_DELIMS = '\\:\\/\\?\\#\\[\\]\\@';
+const SUB_DELIMS = "\\!\\$\\&\\'\\(\\)\\*\\+\\,\\;\\=";
```
Comment (agaripian) [inline src/uri.js]: how is this one passing? is it because of `'` being used in the middle of the string?

Comment (weikinhuang) [inline src/uri.js]: yes http://eslint.org/docs/rules/quotes

**Comment by agaripian** [comment]
👍

**Comment by kaicataldo** [comment]
:+1:

**Pull request merged**